_schemaV2.graphql

```diff
@@ -10913,13 +10932,20 @@ type HomeView {
 
 # A component specification
 type HomeViewComponent {
+  # A background color for this section
+  backgroundColor: String
+
+  # A display subtitle for this section
+  subtitle: String
```

Comment:
What do you think about moving these new fields to the new section type, as they are only relevant for this section? I suppose it might be confusing if they're also available for all other sections.